package/yast2-rear.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Sep  7 13:46:49 UTC 2021 - David Diaz <dgonzalez@suse.com>
+
+- Build package only for supported architectures (bsc#1189646).
```

Comment:
*the* package (?)

Reply:
Funny enough: I put _the_, but I removed in the very last second :)